### PR TITLE
Allow skip bad entries logging during import

### PR DESCRIPTION
**Pull request by MishaDemianenko**
Introduce new import tool option: `skip-bad-entries-logging` that will allow to skip logging of all bad entries.
Default value of new option: false

### CODE REVIEW

File: community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java [hunk 1]
```diff
@@ -177,25 +178,30 @@
                 "Number of bad entries before the import is considered failed. This tolerance threshold is "
                         + "about relationships refering to missing nodes. Format errors in input data are "
                         + "still treated as errors" ),
+        SKIP_BAD_ENTRIES_LOGGING ("skip-bad-entries-logging", Boolean.FALSE,
+                "<true/false>", "Whether or not log bad entries detected during import."),
```
Comment (tinwelint) [inline community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java]: Whether or not `to skip` log`ging` bad entries detected during import

**Review by tinwelint — APPROVED** [review]
Great that you also deferred the message string creation, which could be expensive.

**Review by tinwelint — CHANGES_REQUESTED** [review]
Oh, the little change in the option description

**Comment by MishaDemianenko** [comment]
@tinwelint updated :)

**Review by tinwelint — APPROVED** [review]

**Pull request merged**